.changelog

```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Updated Client Config and Install Guides with aarch64 support for 
```

Comment:
Use the same format as the previous entry :-)

Seems you decided to use that, instead of doing as we do for the code and then with tito.